File: week1/pset1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //Pyramid Height:
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);
    
    for (int i = 0; i < n; i++)
    {   
        //Print spaces
        for (int s = i; s < n - 1; s++)
        {
            printf(" ");
        }

        //print hashes
        for (int x = 0; x <= i; x++)
        {   
           
            printf("#");
        }
        //print a new line
        printf("\n");
       

    }
}

File: week6/pset6/sentimental-mario-less/mario.py
# Prints a right-aligned pyramid with known height

# Import get_int function from cs50 library that will be used to ask user for pyramid height
from cs50 import get_int

# Ask user for the pyramid height
while True:
    height = get_int("Height: ")

    # Make sure height is beween 1 and 8
    if height >= 1 and height <= 8:
        break

# Iterate over each row in pyramid
for row in range(height):

    # Print spaces
    for spaces in range(height, 1, -1):
        print(' ', end='')

    # Print hashes
    for hashes in range(row+1):
        print('#', end='')

    # Print new line
    print()

    # Update height of remaining part of pyramid
    height -= 1
